action.yml

```diff
@@ -0,0 +1,42 @@
+name: 'Install and Setup sccache'
+description: 'Install and setup sccache'
+branding:
+  icon: 'arrow-up'
+  color: 'green'
+inputs:
+  key:
+    description: 'Cache key to differentiate between caches'
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+
+    - name: Install sccache Windows
```

Comment:
What about Linux? It's already installed?

Reply:
yes, and see https://github.com/f3d-app/sccache-setup-action/issues/5